src/cljs/rems/spa.cljs

```diff
@@ -237,6 +239,14 @@
      [:div.container.main-content [content]]
      [footer]]))
 
+(reg-event-fx
+ :after-translations-are-loaded
+ (fn [{:keys [db]} [_ on-loaded]]
+   (if (seq (:translations db))
+     (on-loaded)
+     (.setTimeout js/window #(dispatch [:after-translations-are-loaded on-loaded]) 100))
+   {}))
```

Comment:
hmm seems like bit of a hack. is there really no way to wait for another event in re-frame?

Reply:
There isn't. Similar problem is also met in the redirection (see `:landing-page-redirect!`).

Reply:
ok